config/smart_answers/check_benefits_financial_support_data.yml

```diff
@@ -345,12 +345,11 @@
       group: Help with your bills
       countries:
         - northern-ireland
-    - name: nhs_low_income_scheme
-      title: NHS Low Income Scheme
-      description: If you have a low income, you may be able to get help with prescriptions, dental care and other health costs.
-      condition: eligible_for_nhs_low_income_scheme?
-      url: https://www.nhs.uk/nhs-services/help-with-health-costs/nhs-low-income-scheme-lis/
-      url_text: Check if you’re eligible for the NHS Low Income Scheme on the NHS website
+    - name: nhs_help_with_health_costs
+      title: NHS Help with health costs
+      description: You may be able to get help with prescriptions, dental care, healthcare travel and other health costs, particularly if you’re on a low income or getting certain benefits.
+      url: https://www.nhs.uk/nhs-services/help-with-health-costs
+      url_text: Check if you’re eligible for help with health costs on the NHS website
```

Comment:
Is this addition meant to be in this commit? It looks like it's introducing a new scheme

Reply:
It's a full replacement/update - the low income scheme is now help with health costs.